lib/axe-injector.js

```diff
@@ -1,8 +1,15 @@
 class AxeInjector {
-  constructor({ driver, config, axeSource = null }) {
+  constructor({ driver, config, axeSource = null, options = {} }) {
     this.driver = driver;
     this.axeSource = axeSource || require('axe-core').source;
     this.config = config ? JSON.stringify(config) : '';
+    this.options = options;
+
+    // default logIframeErrors to true so it retains the original behavior
+    this.options.logIframeErrors =
+      typeof this.options.logIframeErrors !== 'undefined'
```

Comment:
Not a big deal, but why do we support non-boolean values here?

```js
new AxeInjector(driver, config, { logIframeErrors: 'sharks and potatoes' })
```

Reply:
There's no type checking, so we support anything thats truthy and falsey?